### ISSUE TITLE
Now saying "Not Found"

### ISSUE THREAD

**Issue by RealityX0** [StevenFlecha/Sendy_listcleaner_mod#3]
Out of the blue, the list cleaner is saying "Not Found" when I try to clean a list. It was working fine before.

Do you think because of the "New Version" Sendy could have deactivated it on older versions? Or am I missing something? I'm using 2.1.2.7

Does changing the name of my main list have any baring on the program's ability to find the dates?

Thank you,

**Comment by KVSocial** [comment]
there is not such error 'not found' so I assume you are talking about 404 not found here.. and when does this happen? Again you leave me guessing here.. if it happens when clicking the 'clean now'  button it means teh file flecha_list_cleaner.php is not there (or issue with your apache/nginx conf with the extension-less php scripts)..

Anyway its not anything that sendy did to the installation (sendy doesn't do anything automatic)

**Comment by RealityX0** [comment]
![image](https://user-images.githubusercontent.com/18557657/29566559-41c77b48-8718-11e7-83a5-46b39ee52511.png)

Not sure if this if 404....This is after I hit the "Clean Now" button.

And when I said "New Version"... I mean because he is forcing this change, making people upgrade to version 3.0 in order to use list cleaner.

Anything towards resolution would be greatly appreciated.
Thank you.

**Comment by RealityX0** [comment]
Ok... since I use Apache, it seems this can be done in the .htaccess file? Been trying to use various fixes with no success. The htaccess is the default used by sendy... so i'm not sure if that is the issue. I added a custom 404 error page, however even after removing the code, clearing cache, and using other computers/browsers, it still has the same behavior. .... Could this be some kind of db issue?

Edit:
After doing more digging, my php error logs say the file isn't being found.

File does not exist: /something/something/something/flecha_list_cleaner, referer: https://....

The name of the file is flecha_list_cleaner.php ... so is this an extension problem?

Edit:
Is flecha_activity_log in the sendy database supposed to be blank?? it seems mine is.

Thank you,

**Comment by KVSocial** [comment]
yes its an apache issue.. however if you haven't properly configured extension-less php files for sendy whole of sendy installation wouldn't be working properly..

no it should not be empty otherwise there's not much the cleaning can do.. but most likely tracking hasn't worked properly bec of same reason.

**Comment by RealityX0** [comment]
Everything else in Sendy is working without issue. Sendy tracking clicks and opens is working fine... any ideas?
